src/device/pf_block_writer.c

```diff
@@ -3990,6 +4003,15 @@ static void pf_put_pd_multiblock_port_and_statistics (
 {
```

Comment:
Check if TODO-comment in function description could be removed

Reply:
I think @olbjo will study this in more detail in the work he is doing on multi ports right now.